src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -260,34 +260,36 @@ const HoloFuelDnaInterface = {
 
       const { transactions } = await createZomeCall('transactions/list_transactions')(params)
       const nonActionableTransactions = transactions.map(presentTransaction).filter(tx => !(tx instanceof Error))
-      const noDuplicateIds = _.uniqBy(nonActionableTransactions, 'id')
-      const presentedCompletedTransactions = await getTxWithCounterparties(noDuplicateIds.filter(tx => tx.status === 'completed'))
+      const uniqueNonActionableTransactions = _.uniqBy(nonActionableTransactions, 'id')
+      const presentedCompletedTransactions = await getTxWithCounterparties(uniqueNonActionableTransactions.filter(tx => tx.status === 'completed'))
       return presentedCompletedTransactions.sort((a, b) => a.timestamp > b.timestamp ? -1 : 1)
     },
     allActionable: async () => {
       const { requests, promises, declined, canceled } = await createZomeCall('transactions/list_pending')()
       const actionableTransactions = await requests.map(r => presentPendingRequest(r)).concat(promises.map(p => presentPendingOffer(p))).concat(declined.map(presentDeclinedTransaction)).concat(canceled.map(presentIncomingCanceledTransaction))
-      const presentedActionableTransactions = await getTxWithCounterparties(actionableTransactions)
+      const uniqActionableTransactions = _.uniqBy(actionableTransactions, 'id')
```

Comment:
Was this taken out at one point?  I thought we deduped all our original endpoints when first creating the interface?  Good find, nonetheless, this *should* be our culprit. :+1: